backend/test/integration/run_tests_locally.sh

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 
 set -e
+set -x
```

Comment:
These two commands can be combined `set -ex`

Reply:
Thank you for catching this! Just fixed it.